addons/project/models/project_task.py

```diff
@@ -1015,6 +1019,10 @@ def create(self, vals_list):
                 task.message_subscribe(follower.partner_id.ids, follower.subtype_ids.ids)
             if current_partner not in task.message_partner_ids:
                 task.message_subscribe(current_partner.ids)
+            if task.email_cc:
+                res = tools.email_split(task.email_cc)
+                partner_ids = self.env['res.partner']._search([('email', 'in', res)])
```

Comment:
Same here as int ENT: maybe we don't need that much variables. But as you wish :)